packages/bot-web-ui/src/stores/journal-store.js

```diff
@@ -10,6 +10,21 @@ export default class JournalStore {
     constructor(root_store) {
         this.root_store = root_store;
         this.dbot = this.root_store.dbot;
+        this.journal_storage_key = 'journal_cache';
+        this.get_journal_messages = JSON.parse(sessionStorage.getItem(this.journal_storage_key));
```

Comment:
Same like `run-panel-store.js`, can we create a function for logic that we're re-using? Referring to `JSON.parse(sessionStorage.getItem(this.journal_storage_key))`.